cpp/include/raft/core/bitset.cuh

```diff
@@ -32,12 +32,41 @@
 
 namespace raft::core {
 
+template <typename index_t>
+_RAFT_HOST_DEVICE void inline compute_original_nbits_position(const index_t original_nbits,
```

Comment:
It looks like the original endianness is not considered, if we have potential restrictions on the application scenario, we should let the caller know via assert or doc.

Reply:
I don't think endianness should be considered in this situation, because the bitset is getting created and used on the same system.